packages/ketcher-react/src/script/ui/views/toolbars/RightToolbar/RightToolbar.module.less

```diff
@@ -19,6 +19,12 @@
 .root {
   .toolbar();
   .vertical-toolbar();
+  position: relative;
+  top: 0;
+  right: 60px;
+  margin-top: 20px;
+  margin-right: 20px;
+  z-index: 10;
 
```

Comment:
Note that this toolbar is nested within a rightmost column of a grid defined in `App.module.less`. Having a `margin-right` causes grid column to bulge, but `right` makes the toolbar render outside of its grid-area:
![image](https://user-images.githubusercontent.com/18420141/159303710-dfa24472-38f6-4b94-8a34-557e26fe06c0.png)

As long as we don't need that column anymore, maybe it'd be better to remove right column from root grid at all, and just position the Elements panel `absolute`ly?

Reply:
removed the right column